mlflow/utils/async_logging/async_logging_queue.py

```diff
@@ -100,9 +143,14 @@ def _log_run_data(self) -> None:
 
         Returns: None
         """
-        run_batch = None  # type: RunBatch
+        async_logging_waiting_time = MLFLOW_ASYNC_LOGGING_WAITING_TIME.get()
+        run_batches = None  # type: RunBatch
```

Comment:
```suggestion
```
nit: Do we need this declaration? It seems the downstream logic assumes non-null.

Reply:
nice catch too, fixed!